File: testdata/fixtures/java-clean/src/test/java/com/example/service/OrderServiceTest.java
// A test source: its imports are attributed as test-only edges (src/test tree).
package com.example.service;

import com.example.domain.Order;

public final class OrderServiceTest {
    public boolean totalIsPreserved() {
        Order order = new OrderService().placeOrder(10);
        return order.total == 10;
    }
}

File: testdata/fixtures/kotlin-clean/src/test/kotlin/com/example/service/OrderServiceTest.kt
// A test source: its imports are attributed as test-only edges (src/test tree).
package com.example.service

import com.example.domain.Order

class OrderServiceTest {
    fun totalIsPreserved(): Boolean {
        val order: Order = OrderService().placeOrder(10)
        return order.total == 10
    }
}
